app/views/partials/_user.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if user.present? %>
+<div class="user-card">
+  <div class="user-photo-box">
+    <img src="<%= user.photo %>" alt="User photo" class="user-photo">
+  </div>
+  <div class="user-details">
+    <h3 class="user-name"><%= user.name %></h3>
+    <small>Number of posts: <%= user.posts.length %></small>
+  </div>
+</div>
+<% else %>
+  <p>User not found.</p>
+<% end %>
```

Comment:
You have done a good job meeting over half of the requirements, however, there is a problem when the UI is opened:

- The root page is still the default rails welcome page [OPTIONAL since it is not in the scope of this milestone]
- When I navigate to the user route, clicking on a user should route me to the specific user's page details
- when they see all post link is clicked on the user show page, the error in the screenshot below occurs

I strongly recommend you look into the above-mentioned issue in order to make the app work seamlessly and to become easily navigatable by links 

<img width="951" alt="image" src="https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/84930355/7bac7469-397c-4d1c-8625-a4723282fee6">